kubernetes/cray-nexus/Chart.yaml

```diff
@@ -41,6 +41,6 @@ dependencies:
 maintainers:
   - name: rnoska-hpe
 icon: https://raw.githubusercontent.com/sonatype/nexus-public/release-3.25.0-03/components/nexus-rapture/src/main/resources/static/rapture/resources/icons/x100/nexus-black.png
-appVersion: 3.25.0-2
+appVersion: 3.25.0
 annotations:
   artifacthub.io/license: MIT
```

Comment:
Bo is right, images used by Helm chart need to be listed via `artifacthub.io/images` annotation. Here's the example: https://github.com/Cray-HPE/cray-spire/blob/master/charts/spire/Chart.yaml#L37-L61

Reply:
Thanks for the information. I was not aware of this.